### ISSUE TITLE
Multilang

### ISSUE THREAD

**Issue by martingronlund** [initiativets/handbook#2]
- english
- swedish

**Comment by martingronlund** [comment]
Scope
=====

How to: https://toolchain.gitbook.com/languages.html

For folder names, let's use [ISO 639-1 Code](https://www.loc.gov/standards/iso639-2/php/code_list.php), e.g. `en` and `sv`.

**Comment by martingronlund** [comment]
I've merged the pull request I did and commited tons of changes since just to make it look OK. The [documentation](https://toolchain.gitbook.com/config.html) seems to be outdated, and the book is not displayed in different languages.

As such, I've just made different groups for Swedish and English for now. They claim they are planning to support multilang again.

Read more in the `V2 changes` documents:
https://docs.gitbook.com/v2-changes/feature-highlights
https://docs.gitbook.com/v2-changes/important-differences

Judging from the [changelog](https://docs.gitbook.com/changelog), V2 seems to be under active development. However, given that the previous months have focused more on user experience improvements and bugfixes, I doubt we will have multilang until end of next year, ish.

**Comment by martingronlund** [comment]
1. I think we should convert back to a single language for now, since GitBook lacks support and it eases the burden on us.
2. I think we should use English as our main language.
	- this makes it accessible to far more people who could be willing to contribute
	- I think our aim should be the world and not just Sweden, as it's silly to limit the scope to a single country. Politics is inherently multinational albeit it's been historically ignored as such, perhaps because of power-mongering and lack of infrastructure to operate internationally. Several initiatives across the globe are putting and have put a change to that and I see no reason for us not to do the same.

**Comment by kristoferlund** [comment]
I agree in principal to both arguments. Let's take in to account where we currently are before pushing forward.

1. Not having to maintain two versions eases our burden. Translating existing information to English on the other hand adds a quite substantial amount of work.

2. I will take this idea to Sanna and discuss how it fits in with the big picture and the overall strategy for IN. My sense is that the benefits clearly outweigh the drawbacks but I am quite certain other opinions exist on the matter - inclusion of elderly etc. 

Let's discuss this further on thursday.
